tests/RestBridge.cpp

```diff
@@ -17,6 +17,37 @@ BOOST_AUTO_TEST_CASE( test_construction )
     BOOST_CHECK_NO_THROW( restbridge::RestBridge( "localhost", 12345 ));
 }
 
+BOOST_AUTO_TEST_CASE( test_construction_argc_argv )
+{
+    const char* argv[] = { "--chocolate", "is-good",
+                           "--something-different...", "is-not-good",
+                           "--restbridge-zeq", "zeq01://localhost:6666",
+                           "--(some-well-known-OS)8", "is-not-really-chocolate" };
+    const int argc = 8;
+    BOOST_CHECK_NO_THROW( restbridge::RestBridge( argc, argv ));
+}
+
+BOOST_AUTO_TEST_CASE( test_construction_argc_argv_invalid )
```

Comment:
Put three separate BOOST_AUTO_TEST_CASE with a proper name so you see easily what each test is testing :)